canonical/ubuntu.com
# adding vwo

## Done

- Added an HTML file of the VWO smartcode to be included as the script for Google Tag Manager
- Added the include statement to the relevant base files

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please make sure you see no ["flicker"](https://help.vwo.com/hc/en-us/articles/360020440754-Why-Do-I-Notice-a-Page-Flicker-When-the-VWO-Test-Page-is-Loading-) after implementing the code

## Issue / Card

[Jira](https://warthogs.atlassian.net/browse/WD-7085)

## Help

[Documentation](https://help.vwo.com/hc/en-us/articles/360019422834-Configuring-the-VWO-SmartCode-for-Your-Website)
